test/make-auto.d/build_custom.bats.in

```diff
@@ -18,7 +18,12 @@ source common.bash  # for ShellCheck; removed by ch-test
             tarball=${tarballs[0]}
             # Convert to SquashFS if needed.
             if [[ $CH_TEST_PACK_FMT = squash* ]]; then
-                ch-convert "$tarball" "${tarball/tar.?z/sqfs}"
+                # janky fix for centos7xz tar permission errors
+                if [[ $tarball = *centos7xz* ]]; then
+                    ch-convert --no-xattrs "$tarball" "${tarball/tar.?z/sqfs}"
```

Comment:
```suggestion
                    ch-convert $xattrs_arg "$tarball" "${tarball/tar.?z/sqfs}"
```
and supporting code will let you have just one `ch-convert` call, so then you don't have to worry about keeping them in sync.